src/main/kotlin/io/bazel/kotlin/plugin/jdeps/JdepsGenExtension.kt

```diff
@@ -8,39 +8,46 @@ import org.jetbrains.kotlin.analyzer.AnalysisResult
 import org.jetbrains.kotlin.config.CompilerConfiguration
 import org.jetbrains.kotlin.container.StorageComponentContainer
 import org.jetbrains.kotlin.container.useInstance
+import org.jetbrains.kotlin.descriptors.CallableDescriptor
 import org.jetbrains.kotlin.descriptors.ClassDescriptor
 import org.jetbrains.kotlin.descriptors.DeclarationDescriptor
 import org.jetbrains.kotlin.descriptors.DeclarationDescriptorWithSource
 import org.jetbrains.kotlin.descriptors.FunctionDescriptor
 import org.jetbrains.kotlin.descriptors.ModuleDescriptor
-import org.jetbrains.kotlin.descriptors.ParameterDescriptor
 import org.jetbrains.kotlin.descriptors.PropertyDescriptor
 import org.jetbrains.kotlin.descriptors.SourceElement
+import org.jetbrains.kotlin.descriptors.ValueDescriptor
 import org.jetbrains.kotlin.descriptors.impl.LocalVariableDescriptor
 import org.jetbrains.kotlin.extensions.StorageComponentContainerContributor
-import org.jetbrains.kotlin.load.java.descriptors.JavaMethodDescriptor
-import org.jetbrains.kotlin.load.java.descriptors.JavaPropertyDescriptor
+import org.jetbrains.kotlin.js.translate.callTranslator.getReturnType
```

Comment:
Is this the right function to be using? It looks like it's being loaded from the javascript package.

Reply:
I remember thinking the same thing when I imported it, but it was needed to access a type which was required to get one of my new tests to pass. Can someone point me to a more appropriate alternative?

Reply:
I briefly looked around and didn't see anything that stands out. If anything you can just inline the function itself, which is just calling into another function with `resolvedCall` as an argument.

Reply:
Thanks for flagging this and taking another look. I just got in front of my computer, and the great news is that I was able to remove the code using that javascript class, and all my tests still pass. I was trying to page back in what I was thinking at the time, and the best I can come up with is that `getReturnType` always returned a KotlinType, while the alternative below that optionally returns a KotlinType. To confirm the new code is functionally equivalent, 
I'll also publish a new internal version of `rules_kotlin` to test in our monorepo, and report back if everything looks good. Thanks again!